randovania/games/fusion/logic_database/Sector 6 NOC.txt

```diff
@@ -19,7 +19,12 @@ Extra - room_id: [0]
   > Door to Geron's Crossing
       Trivial
   > Door to Glass Tube to Sector 4 (AQA)
-      Screw Attack and Can Break Single Bomb Blocks
+      All of the following:
+          Screw Attack
+          Any of the following:
+              Can Break Single Bomb Blocks
+              # Currently logically useless: but maybe if we have an option to remove screw blocks https://youtu.be/Ifde8nknvYk
+              Level 0 Locks and Speed Booster and Shinespark Tricks (Advanced) and Disabled Door Lock Rando and Disabled Entrance Rando
```

Comment:
missing knowledge intermediate, since the bomb+speed interaction isnt that known.